### ISSUE TITLE
Server responses should adhere to same data type locally and online

### ISSUE THREAD

**Issue by Andrej123** [cocoaheads-slovenija/theClocker#20]
There's a discrepancy between server responses weather it's run locally (discrepancy spotted on `macOS 10.12.2 Sierra`) vs. the server that's running online (`clocker.goranche.net`).

Sample for **local** response for `getClients`:
`[{
    "id": "1",
    "name": "AndCli"
}, {
    "id": "2",
    "name": "And2Cli"
}, {
    "id": "3",
    "name": "And3Cli"
}, {
    "id": "4",
    "name": "Andrej"
}, {
    "id": "5",
    "name": "And"
}]`

Sample for **online** response for `getClients`:
`[{
    "id": 1,
    "name": "Someone who pays me"
}, {
    "id": 2,
    "name": "Sponsor"
}, {
    "id": 3,
    "name": "Andrej"
}]`

**Comment by goranche** [comment]
🤔 it might be easier (and cleaner?) to handle this in the client...

**Comment by Andrej123** [comment]
Yeah, I definitely agree that it might be easier. I'm not interested in "easier". I'd like to learn something new. I agree that it might be cleaner, and that's one thing I'll keep in mind.

I'd like to at least give it a try to solve the issue on the server side. If I won't be able to progress fast enough, or the solution would be indeed be coming out messy I'll opt for the client side solution.

**Comment by goranche** [comment]
👍 knock yourself out...
